packages/paste-website/src/pages/components/example-text/index.mdx

```diff
@@ -0,0 +1,112 @@
+  const feature = await getFeature('Example Text');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+      mdxHeadings,
+      pageHeaderData: {
+        categoryRoute: SidebarCategoryRoutes.COMPONENTS,
+        githubUrl: 'https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/example-text',
+        storybookUrl: '/?path=/story/components-example-text--default',
+      },
+    },
+  };
+};
+
+<LivePreview
+  scope={{ExampleText, Paragraph}}
+  language="jsx"
+>
+  {`<>
+    <Paragraph>When submitting feedback about your experience, provide specific details to help us understand your needs and improve our offerings. For example:</Paragraph>
+    <ExampleText>The service was quick and efficient, but I would like more options in the menu.</ExampleText>
+</>`}
+</LivePreview>
+
+## Guidelines
+
+### About Example Text
+
+Example Text represents contextual examples that guide users on what type of information to enter or how to respond within a system. 
+
+Use Example Text so users can easily differentiate between examples and regular text or code blocks. It also reduces the chances of users overlooking important examples.
+
+### How to decide which components to use to make your text stand out
+
+Component       | Usage | Example
+----------------|-------|-----
+Example Text | Use when you want to provide contextual examples of user input. | <ExampleText>This is an Example Text that guides users on how to respond to a system</ExampleText>
+[Blockquote](/components/blockquote) | Use to highlight quotations from an external source or customer feedback. | <Blockquote><BlockquoteContent>This is a Blockquote pulled from an outside source.</BlockquoteContent><BlockquoteCitation author="Google" /></Blockquote>
+[Code Block](/components/code-block) | Use when you need to display blocks of code and when the user needs to copy a command to put it into a CLI. | <CodeBlock variant="single-line" code={`const accountSID = process.env.TWILIO_ACCOUNT_SID`}/>
+[Inline Code](/components/inline-code) | Use to emphasize short, inline pieces of computer code such as variable names and function calls. | <InlineCode>console.log(mySustainabilityGoals)</InlineCode>
+[Help Text](/components/help-text) | Use below a form field to help users prevent an error and describe what makes the form field successful. | <><Label htmlFor="email_address" required>Email address</Label><Input aria-describedby="email_help_text" id="email_address" name="email_address" type="email" placeholder="example@twilio.com" onChange={()=>{}} required/><HelpText id="email_help_text">Use your work email address.</HelpText></>
+[Callout](/components/callout) | Use a Callout to highlight any other important information. | <Callout variant='neutral'><CalloutHeading as="h2">Heads up!</CalloutHeading><CalloutText>This is some information you need to know.</CalloutText></Callout>
```

Comment:
question: If callout is included in the list why would alert not also be there? Or is it that alert has a very explicit use case and less chance of confusion?

Reply:
this is a question for @jhanvi1399 !

Reply:
Since a callout is used to highlight specific information, there is a slighttt overlap in use cases between blockquotes and callouts. Additionally, our previous visual design for blockquotes resembled a neutral callout, which is why I included the callout here.

Reply:
Thank you for confirming! I'm happy and marking as resolved